frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -79,35 +84,20 @@ const Map: React.FC<MapProps> = ({
   const [baseLayers, setBaseLayers] = useState<BaseLayer[]>([]);
   const [activeBasemap, setActiveBasemap] = useState<BaseLayer | null>(null);
 
-  // different markers for building and parcel
-  var greenIcon = new Icon({
-    iconUrl: require('assets/images/marker-icon-2x-green.png'),
-    shadowUrl: require('assets/images/marker-shadow.png'),
-    iconSize: [25, 41],
-    iconAnchor: [12, 41],
-    popupAnchor: [1, -34],
-    shadowSize: [41, 41],
-  });
-
-  var blueIcon = new Icon({
-    iconUrl: require('assets/images/marker-icon-2x-blue.png'),
-    shadowUrl: require('assets/images/marker-shadow.png'),
-    iconSize: [25, 41],
-    iconAnchor: [12, 41],
-    popupAnchor: [1, -34],
-    shadowSize: [41, 41],
-  });
-
   //do not jump to map coordinates if we have an existing map but no parcel details.
   if (mapRef.current && !selectedProperty?.parcelDetail) {
     lat = (mapRef.current.props.center as Array<number>)[0];
     lng = (mapRef.current.props.center as Array<number>)[1];
   }
-  const lastZoom = useSelector<RootState, number>(state => state.mapViewZoom) ?? zoom;
+  const lastZoom = useSelector<RootState, number>(state => state.mapViewZoom) ?? zoomProp;
   useEffect(() => {
     dispatch(resetMapViewZoom());
   }, [dispatch]);
 
+  // TODO: refactor various zoom settings
+  const [bounds, setBounds] = useState<BBox>();
+  const [zoom, setZoom] = useState(lastZoom);
+
```

Comment:
As mentioned in the TODO we have a few ways in which we keep track of the zoom level of the map. This needs to be refactored in a future PR to consolidate into one single approach